docs/MigrationGuides.md

```diff
@@ -0,0 +1,15 @@
+# Migration Guides
+
+This project follows [Semantic Versioning](http://semver.org).
+
+Please follow the appropriate guide below when **upgrading to a new major version** of Moya (e.g. 8.0 -> 9.0).
+
+## Upgrade from 8.x to 9.x
+
+- Move the `parameters` and `parameterEncoding` to the `task` computed property by using the case `.requestParameters(parameters:,encoding:)`
```

Comment:
Another stylistic one, signatures are rather references as:
```swift
.requestParameters(parameters:encoding:)
```

instead of:
```swift
.requestParameters(parameters:,encoding:)
```